src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -24,7 +29,7 @@ private List<Product> getAllProductsFromShoppingCart(Long userId) {
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
-        // TODO: add log message about successful fetched data from DB
+        logger.info("Data from DB fetched successfully for userId: {}", userId);
```

Comment:
The log message in the `getAllProductsFromShoppingCart` method should be more informative. It's good practice to include details about the operation that was performed, such as the number of products fetched.